python/tvm/auto_scheduler/task_scheduler.py

```diff
@@ -75,7 +76,7 @@ def make_search_policies(
         if model_type == "xgb":
             cost_model = XGBModel(num_warmup_sample=len(tasks) * num_measures_per_round)
             if load_model_file:
-                logger.info("Load pretrained model...")
+                logger.info("TaskScheduler: Load pretrained model...")
                 cost_model.load(load_model_file)
             elif load_log_file:
                 cost_model.load_log_file(load_log_file)
```

Comment:
Wrong function name:

```suggestion
                cost_model.update_from_file(load_log_file)
```